opendatahub-io/data-science-pipelines-operator
# V2 tests

## The issue resolved by this Pull Request:
Resolves https://issues.redhat.com/browse/RHOAIENG-2581

## Testing instructions
see kind tests run in gh workflows ui
or try running them in your local cluster [here](https://github.com/opendatahub-io/data-science-pipelines-operator/tree/main/tests#run-tests-locally) 

## Checklist
